de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/IHUPackingMaterialsCollector.java

```diff
@@ -155,4 +155,8 @@ void addLU(I_M_HU luHU,
 	 */
 	IHUPackingMaterialsCollector<T> setProductIdSortComparator(Comparator<Integer> productIdsSortComparator);
 
+	void setisCollectTUNumberPerOrigin(boolean b);
```

Comment:
what is this method about? also imho it shoul be named `setCollectTUNumberPerOrigin` (->javadoc)